apps/src/code-studio/components/progress/StageLockDialog.jsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react';
+import Radium from 'radium';
+import { connect } from 'react-redux';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
```

Comment:
Is there a particular reason you're using `BaseDialog` and not the more opinionated `Dialog` @pcardune recently introduced?